webapp/channels/src/components/quick_switch_modal/quick_switch_modal.tsx

```diff
@@ -157,7 +157,7 @@ export default class QuickSwitchModal extends React.PureComponent<Props, State>
         const providers: SwitchChannelProvider[] = this.channelProviders;
 
         const header = (
-            <h1>
+            <h1 id='quickSwitchHeader'>
```

Comment:
curious why we had to do this change?

Reply:
This is the main title of the modal/dialog. Doing so gives the dialog an accessible name by referring to the element that provides the dialog title (by `aria-labelledby='quickSwitchHeader'`). 

Meanwhile, the dialog description (by `aria-describedby='quickSwitchHeaderWithHint'`) will read out as is with title + hint.

Reply:
🤦🏽 oh no i didnt checked the aria-labelledby. Sorry